runners/samza/src/main/java/org/apache/beam/runners/samza/SamzaRunner.java

```diff
@@ -82,11 +84,15 @@ private SamzaRunner(SamzaPipelineOptions options) {
 
   public PortablePipelineResult runPortablePipeline(RunnerApi.Pipeline pipeline, JobInfo jobInfo) {
     final String dotGraph = PipelineDotRenderer.toDotString(pipeline);
-    LOG.info("Portable pipeline to run:\n{}", dotGraph);
+    LOG.info("Portable pipeline to run dot graph:\n{}", dotGraph);
```

Comment:
why do we want both dot graph and json graph?

Reply:
We just need the json graph. We don't need both. I will remove the dot graph.